lib/shop/shop_page.dart

```diff
@@ -1,5 +1,8 @@
 import 'package:auto_route/auto_route.dart';
+import 'package:flex_storefront/category_list/cubits/category_cubit.dart';
+import 'package:flex_storefront/category_list/cubits/category_state.dart';
```

Comment:
Do we want to export these in a single file? i.e. `category_cubit.dart` or always keep separate?

Reply:
I would go with the separated imports, barrel files are making my dev workflow harder TBH.